Yoast/wordpress-seo
# Add passive voice assessment for Farsi

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Implements the passive voice assessment for Farsi.
* Adds Passive voice assessment for Farsi.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Enable the `FARSI_SUPPORT` feature by adding `define( 'YOAST_SEO_FARSI_SUPPORT', true );` in `basic-wordpress-config.php` file
* Set your website to Farsi (فارسى).
* Test this functionality in Yoast SEO Free.
* Create a new empty post and add these sentences to your post:
`.او ربوده شد`
`.غذا رفته است`
`.امپراطوری زمین خورده`
`.دانش آموز خبردار شده`
`.جاده آسفالت شده`
`.تصاویر آراسته بودند`

* Check that the passive voice bullet turns red and all sentences are highlighted.

* Add these sentences and check that that they do not get highlighted:
`.آنها دوچرخه را تعمیر می کنند`
`.باران شروع شد`
* Disable the `FARSI_SUPPORT` feature by adding `define( 'YOAST_SEO_FARSI_SUPPORT', false );` in `basic-wordpress-config.php` file

* Save the changes in the post

* Reload the page

* Confirm that the Passive voice assessment doesn't appear in the metabox

### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

*

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/secure/RapidBoard.jspa?rapidView=99&modal=detail&selectedIssue=LINGO-926
